deploy/gk-deploy

```diff
@@ -151,13 +151,15 @@ abort() {
     eval_output "${CLI} delete template deploy-heketi 2>&1"
     eval_output "${CLI} delete template heketi 2>&1"
   else
+    eval_output "${CLI} delete clusterrolebinding heketi-sa-view 2>&1"
```

Comment:
it removes a clusterrolebinding that was added in deployment, this was needed to be added in order for gk-deploy to work in kube 1.6, which uses RBAC authorization instead of ABAC.
it was added in PR #236